README.md

```diff
@@ -62,7 +70,7 @@ APP_SCOPES=openid,phone,profile,email
 
 Now, you can run `docker-compose run` if you use Docker. `docker-compose.yml` contains DB user/password information.
 
-4. Create your database and configure database connection. This service uses PostgreSQL as default database, so if you use PostgreSQL, just add your connection `DATABASE_URL=....` in your `.env`. If you want to test how it works, please run PostgreSQL in your machine with a new database (`$ createdb skeleton_nodejs`) and proceed with the next steps. Or else you can use `docker-compose run`, which comes with PostgreSQL.
+4. Create your database and configure database connection. This service uses PostgreSQL as default database, so if you use PostgreSQL, just add your connection `DATABASE_NAME=....` in your `.env`. If you want to test how it works, please run PostgreSQL in your machine with a new database (`$ createdb skeleton_nodejs`) and proceed with the next steps. Or else you can use `docker-compose run`, which comes with PostgreSQL.
```

Comment:
Not only DATABASE_NAME, right? It's misleading

Reply:
yes also name, port, user, password. I put the ... there as etc...